CHANGELOG.md

```diff
@@ -560,6 +560,7 @@
 - [Profile engine startup][4110]
 - [Report type of polyglot values][4111]
 - [Engine can now recover from serialization failures][5591]
+- [Update to GraalVM 22.3.1][????]
```

Comment:
```suggestion
- [Update to GraalVM 22.3.1][5602]
```

Reply:
Wow, CI job that checks that have not found that I had `???`. I guess that this was my bad. Next time I will remember to not touch the CHANGELOG until I have a PR. Thanks for noticing this.

Reply:
I thought that the CI job was only checking if changelog was modified at all, I don't think it has any special logic.